### ISSUE TITLE
column width adjust

### ISSUE THREAD

**Issue by bobmahalo** [WhyDRS/website#11]
aloha, not a pressing issue, mores just trying to play with cash instead of paper trading.

the columns in the display are fixed. I tried to make them more narrow to fit more columns on my screen, but haven't figured out how.
similar to a excel spreadsheet they columns can be narrowed to concentrate on other columns.
I did see I could remove entire columns, maybe that is what that feature is there.

**Comment by tehchives** [comment]
Hey Bob, is this in reference to the WhyDRS Database found at database.whydrs.org ?

**Comment by bobmahalo** [comment]
> Hey Bob, is this in reference to the WhyDRS Database found at database.whydrs.org ?

aloha chives. yes. after JB shared the hydras page, I wanted to look at the issuer and 2 other columns, but couldn't make the columns more narrow. I then found the column selection table on the side and was able to remove/add columns as needed. there was just a lot of space on some columns probably due to a row with more info and stretching the width of the entire column?
like I said, It was more of playing with this place than an actual issue.
please disregard.

**Comment by tehchives** [comment]
Gotcha! We can close the issue but I'll tag @JamesAlfonse first so he can chime in. Some ideas from what you mentioned - Maybe there is a way to make the width more dynamic, or force text wrapping to standardize column width. 

If those end up being paths worth exploring a new issue could be opened on the Database repo.

**Comment by JamesAlfonse** [comment]
Yes, I mention the issue [here](https://github.com/WhyDRS/Database/issues/17). Absolutely agree that there should be a way to adjust the column size or have the columns shrink when a column is removed. Thanks for bringing it to our attention @bobmahalo ! @tehchives

**Comment by JamesAlfonse** [comment]
Transferring this to the Database repository, as it related to the table for the database.
